ImperatorToCk2/src/main/java/ImperatorToCK2/Output.java

```diff
@@ -1139,6 +1139,92 @@ public static void copyFlag(String ck2Dir, String modDirectory, String rank, Str
 
     }
     
+    public static int generateFlag(String ck2Dir, String irDirectory, String rank, ArrayList<String[]> flagList, String tag, String flagID,
```

Comment:
Return a boolean instead of an int.

Reply:
I usually prefer working with integers instead of booleans because there's more that can be stored with an integer then a boolean. 

While right now it only returns a 0 or a 1, what if in the future I need to return a third or a fourth value for certain edge cases? With how wonky CK2 modding and I:R flag syntax script can get, I never know what weird workaround I'll need in the future, and keeping it an integer'll make the code easier to adapt for those undiscovered edge cases